Week-7/Exercise7.2.js

```diff
@@ -0,0 +1,35 @@
+let arr =  [ [ 1, 2, 3 ], 
+             [ 4, 5, 6 ], 
+             [ 7, 8, 9 ] ];
+let N = arr.length, top = 0, down = N-1, left = 0, right = N-1, dir = 0;
+
+while(top <= down && left <= right){
+    if(dir == 0){
+        for(let i=left; i<=right; i++ ){
+            console.log(arr[top][i])
```

Comment:
why are you consoling every where, you have to declare the array and then update the array in every travesal and then at the end print that array,
do not use this console thing every, will not be recommended or allowed in interview